example/dummy/persistent_dummy.go

```diff
@@ -86,18 +89,22 @@ func (app *PersistentDummyApplication) CheckTx(tx []byte) types.Result {
 }
 
 func (app *PersistentDummyApplication) Commit() types.Result {
-	// Save
-	appHash := app.app.state.Save()
-	app.logger.Info("Saved state", "root", appHash)
+	app.height = app.blockHeader.Height
```

Comment:
shouldn't we do this after we successfully saved a new version?

Reply:
I guess so, but if save fails, we panic, and this is only stored locally, so it doesn't really matter.
But sure, I'll clean it up.